source/common/http/filter_manager.cc

```diff
@@ -456,6 +456,14 @@ void ActiveStreamDecoderFilter::sendLocalReply(
     Code code, absl::string_view body,
     std::function<void(ResponseHeaderMap& headers)> modify_headers,
     const absl::optional<Grpc::Status::GrpcStatus> grpc_status, absl::string_view details) {
+
+  this->streamInfo().filterState()->setData(
+      FS_LOCAL_REPLAY_KEY,
+      std::make_shared<LocalReplyOwnerType>(this->filter_context_.filter_name,
+                                            this->filter_context_.config_name,
+                                            std::string(details)),
+      StreamInfo::FilterState::StateType::Mutable, StreamInfo::FilterState::LifeSpan::FilterChain);
```

Comment:
Basically, we keep the `filter_name` here just for backward compatibility and it maybe removed from the filter context in the future. So, it's better to use `config_name` only here.

Reply:
how can I know the extension type? details has some meaning depending the extension, for example ext_authz, it may contains server error, or token invalid

Reply:
Basically, the config_name should be unique in the filter chain. So, the config_name should be enough to tell the related filter type info.